<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::RemoteVideoDialog</class>
 <widget class="QDialog" name="ns::RemoteVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Remote Video</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetOuter {
	border: 1px solid #17242D;
	border-radius: 4px;
	background: transparent;
}

#widgetInner {
	background: transparent;
	border-radius: 4px;
}

/* css of caption */
#frameCaption {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #0078c7, stop: 1.0 #016aae);
/*	background: #555555;*/
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	/*border-top: 1px solid #57A3D3;
	border-left: 1px solid #57A3D3;
	border-right: 1px solid #57A3D3;
    border-bottom: 1px solid #015c97;*/
}

#frameCaption #labelCaptionLogo {
	background-image: url(:/images/ui/logo.png);
	background-repeat: none;
	background-position: center;
	max-width: 32px;
	min-width: 32px;
	max-height: 32px;
	min-height: 32px;
}

#frameCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #ffffff;
}

#frameCaption #frameSystemMenu {
	border: 0px;
}

#frameCaption #frameSystemMenu #sysPanelContainer {
	border: 0px;
}

#frameCaption #pushButtonMaxmize {
	border-image: url(:/images/ui/remote_video_maxmize.png);
}

#frameCaption #pushButtonMaxmize:hover {
	border-image: url(:/images/ui/remote_video_maxmize_hover.png);
}

#frameCaption #pushButtonMaxmize:pressed {
	border-image: url(:/images/ui/remote_video_maxmize_pressed.png);
}

#frameCaption #pushButtonNormal {
	border-image: url(:/images/ui/remote_video_minmize.png);
}

#frameCaption #pushButtonNormal:hover {
	border-image: url(:/images/ui/remote_video_minmize_hover.png);
}

#frameCaption #pushButtonNormal:pressed {
	border-image: url(:/images/ui/remote_video_minmize_pressed.png);
}

#frameCaption #pushButtonMinimize {
	border-image: url(:/images/ui/caption_minimize.png);
}

#frameCaption #pushButtonMinimize:hover {
	border-image: url(:/images/ui/caption_minimize_hover.png);
}

#frameCaption #pushButtonMinimize:pressed {
	border-image: url(:/images/ui/caption_minimize_pressed.png);
}

#frameCaption #pushButtonClose {
	border-image: url(:/images/ui/caption_close.png);
}

#frameCaption #pushButtonClose:hover {
	border-image: url(:/images/ui/caption_close_hover.png);
}

#frameCaption #pushButtonClose:pressed {
	border-image: url(:/images/ui/caption_close_pressed.png);
}

/* end of css caption */

#aboutPanel {
	border: 1px solid #BDE0F4;
	border-radius: 4px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #9BD0F0, stop: 0.2 #EFF8FF);
}

#labelRemoteVideo {
	background: #000000;
}

#widgetRemoteVideo {
	background: #343536;
}

#widgetBottom {
	background: rgba(36,37,38,30%);
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

#widgetBottom #labelDuration {
	color: white;
}

#widgetBottom #widgetNetwork QLabel {
	background-image: url(:/images/ui/vs_network_disable.png);
}

#widgetBottom #pushButtonSetting {
	border-image: url(:/images/ui/vs_setting.png);
}

#widgetBottom #pushButtonSetting:hover {
	border-image: url(:/images/ui/vs_setting_hover.png);
}

#widgetBottom #pushButtonSetting:pressed {
	border-image: url(:/images/ui/vs_setting_pressed.png);
}

#widgetBottom #pushButtonMax {
	border-image: url(:/images/ui/vs_max.png);
}

#widgetBottom #pushButtonMax:hover {
	border-image: url(:/images/ui/vs_max_hover.png);
}

#widgetBottom #pushButtonMax:pressed {
	border-image: url(:/images/ui/vs_max_pressed.png);
}

#widgetBottom #pushButtonMin {
	border-image: url(:/images/ui/vs_min.png);
}

#widgetBottom #pushButtonMin:hover {
	border-image: url(:/images/ui/vs_min_hover.png);
}

#widgetBottom #pushButtonMin:pressed {
	border-image: url(:/images/ui/vs_min_pressed.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetOuter" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetInner" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frameCaption">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>31</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelCaptionLogo">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTitle">
              <property name="text">
               <string>YesPo Service Client</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>68</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="sysPanelContainer">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="sysPanel" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButtonMinimize">
                    <property name="minimumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonMaxmize">
                    <property name="minimumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonNormal">
                    <property name="minimumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonClose">
                    <property name="minimumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>35</width>
                      <height>36</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="autoDefault">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>9</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetRemoteVideo" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="ns::VideoLabel" name="labelRemoteVideo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetBottom" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QWidget" name="widgetDuration" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>9</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelDuration">
                 <property name="text">
                  <string>00:00:00</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetNetwork" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelNetwork1">
                 <property name="minimumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelNetwork2">
                 <property name="minimumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelNetwork3">
                 <property name="minimumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelNetwork4">
                 <property name="minimumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelNetwork5">
                 <property name="minimumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>8</width>
                   <height>8</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>32</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButtonSetting">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMax">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMin">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::VideoLabel</class>
   <extends>QLabel</extends>
   <header>videolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ns::RemoteVideoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinimize</sender>
   <signal>clicked()</signal>
   <receiver>ns::RemoteVideoDialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
